src/api/app/instrumentations/workflow_step_instrumentation.rb

```diff
@@ -0,0 +1,16 @@
+module WorkflowStepInstrumentation
+  extend ActiveSupport::Concern
+
+  # Define callbacks with ActiveModel::Callback which is included in ActiveModel::Model
+  included do
+    define_model_callbacks :initialize
+
+    after_initialize :track_instantiation
+  end
+
+  private
+
+  def track_instantiation
+    RabbitmqBus.send_to_bus('metrics', "workflow_step.#{self.class.name.demodulize.underscore}.instantiation value=1")
```

Comment:
With this data, I would like to have a panel showing how much each step is instantiated. I wonder if this panel would always need to be updated whenever we introduce a new workflow step. I would like to avoid this. If I change this line to the following, I could group by value, right?

```suggestion
    RabbitmqBus.send_to_bus('metrics', "workflow_step.instantiation value=#{self.class.name.demodulize.underscore}")
```